package.json

```diff
@@ -18,6 +18,10 @@
   },
   "homepage": "https://github.com/Marlyn-Mayienga/Awesome_Books#readme",
   "devDependencies": {
-    "hint": "^7.1.1"
+    "hint": "^7.1.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
   }
 }
```

Comment:
### Your MIT
![MIT-empty](https://user-images.githubusercontent.com/93039359/182528044-a1949609-fa37-459c-a6b8-049e986ca580.png)

### Example MIT
![example](https://user-images.githubusercontent.com/93039359/182528100-9867f6be-2c08-4b3d-9cda-63a919a03387.png)


- [ ] Your MIT.md file is empty because it doesn't contain **your name** and your **partner name**. It doesn't contain **current year** and **APP-TYPE**. Please add these missing things :+1: It is your license file to please give attention to this :thinking: